tests/e2e/csi_snapshot_basic.go

```diff
@@ -4119,6 +4132,23 @@ var _ = ginkgo.Describe("[block-vanilla-snapshot] Volume Snapshot Basic Test", f
 		// After reboot.
 		bootstrap()
 
+		fullSyncWaitTime := 0
+
+		if os.Getenv(envFullSyncWaitTime) != "" {
+			fullSyncWaitTime, err = strconv.Atoi(os.Getenv(envFullSyncWaitTime))
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			// Full sync interval can be 1 min at minimum so full sync wait time
+			// has to be more than 120s.
+			if fullSyncWaitTime < 120 || fullSyncWaitTime > defaultFullSyncWaitTime {
+				framework.Failf("The FullSync Wait time %v is not set correctly", fullSyncWaitTime)
+			}
+		} else {
+			fullSyncWaitTime = defaultFullSyncWaitTime
+		}
+
+		ginkgo.By(fmt.Sprintf("Double Sleeping for %v seconds to allow full sync finish", fullSyncWaitTime))
```

Comment:
`fullSyncWaitTime` is more double the full sync interval why are you doubling it again

Reply:
Based on a PR report and investigation done for the new change of config secret, 
===============
Earlier, after the VC reboot was successfully performed, the PVC was able to bind with the e2e test standards of 5 min
now, the PVC is taking a little longer time (~6 to ~8 min) - this needs code modification to run this test regularly on the BR pipeline.
================
So have added a double sleep to recover the system